platforms/core-runtime/wrapper-main/src/integTest/groovy/org/gradle/integtests/WrapperGenerationIntegrationTest.groovy

```diff
@@ -35,7 +35,7 @@ import java.util.jar.Manifest
 import static org.hamcrest.CoreMatchers.containsString
 
 class WrapperGenerationIntegrationTest extends AbstractIntegrationSpec {
-    private static final HashCode EXPECTED_WRAPPER_JAR_HASH = HashCode.fromString("498495120a03b9a6ab5d155f5de3c8f0d986a449153702fb80fc80e134484f17")
+    private static final HashCode EXPECTED_WRAPPER_JAR_HASH = HashCode.fromString("76713a824621c02c1cdae01ee6aca6aa31dca363668e6af5b40036e8960640f3")
 
```

Comment:
Hi @ov7a could you explain why we need this? 🙇

Reply:
In short, this is a test that ensures that the wrapper jar is always reproducible, meaning the same jar is emitted if there are no changes to the wrapper code. However, the downside of that test is that this hardcoded checksum needs to be updated every time the wrapper code is changed.

See also
* https://github.com/gradle/gradle/issues/29718

Reply:
Thanks @ov7a